.github/workflows/build.yml

```
@@ -333,7 +333,7 @@ jobs:
         --build-arg STELLAR_CORE_IMAGE_REF=stellar-core:${{ inputs.arch }}
         --build-arg HORIZON_IMAGE_REF=stellar-horizon:${{ inputs.arch }}
         --build-arg FRIENDBOT_IMAGE_REF=stellar-friendbot:${{ inputs.arch }}
-        --build-arg SOROBAN_RPC_IMAGE_REF=stellar-soroban-rpc:${{ inputs.arch }}
+        --build-arg STELLAR_RPC_IMAGE_REF=stellar-stellar-rpc:${{ inputs.arch }}
```

We should reduce this to `stellar-rpc`. Each app just needs the `stellar-` prefix once. Before because the app was named `soroban-rpc` we added the `stellar-` prefix, but now because the `stellar-rpc` already has that in its name, we don't need to have it double.
```suggestion
        --build-arg STELLAR_RPC_IMAGE_REF=stellar-rpc:${{ inputs.arch }}
```